Pear/Onboarding/OnboardingPageViewController.swift

```diff
@@ -45,7 +45,6 @@ class OnboardingPageViewController: UIPageViewController {
 
         let demographicsViewController = DemographicsViewController(delegate: self)
         let interestsViewController = InterestsViewController(delegate: self)
-        let talkingPointsViewController = TalkingPointsViewController(delegate: self)
```

Comment:
okay here it looks like we've committed to removing talking points so it's just a matter of purging anything related to it throughout the codebase after everything else has been properly integrated